spec/policies/task_policy_spec.rb

```diff
@@ -24,20 +24,21 @@
   describe '#create?' do
     subject { policy.create? }
 
-    let(:creator) { User.new(id: 43) }
-    let(:project) { Project.new(id: 42, user: creator) }
-    let(:task) { Task.new(project: project) }
+    let(:project) { create :project, user_id: creator }
+    let(:task) { create :task, project: project, user_id: user }
```

Comment:
Why you need it at all? Your `create?` method is: https://github.com/Amir-Gaifullin/task-tracker/blob/9e034183583e0d8de677ec391d2c392d76258bbe/app/policies/task_policy.rb#L8-L10